packages/big-design/src/components/Panel/Panel.tsx

```diff
@@ -40,10 +40,12 @@ export const RawPanel: React.FC<PanelProps & PrivateProps> = memo(({ forwardedRe
       return null;
     }
 
+    const headerBottomMargin = description ? 'xxSmall' : props.children ? 'medium' : 'none';
```

Comment:
I don't love a nested ternary, but this is just an A or B or C evaluation.

Happy to move it to a `let` and `if` combo (or a better recommendation) if people prefer.

Reply:
🤔 I wonder if something like `gap` will work better in this scenario

Reply:
I support `gap` as well but this is fine, up to you 🍹 . In hindsight it is what I would use but `gap` wasn't readily available yet.

Reply:
`gap` might be over egging it 🤷 

We just have the one bit of spacing here, albeit with multiple possible values
Whereas a `gap` infers a more complex, multi row/column layout, where the value is intended to create multiple spaces

or it could just be that I'm old and don't like new CSS 😆

Reply:
@icatalina I've gone for the approach you mentioned here: https://github.com/bigcommerce/big-design/pull/1429/commits/58c3a28b615f316e4bfacbe26b4436204205d85d